northstar/src/runtime/config.rs

```diff
@@ -17,6 +17,29 @@ use std::{collections::HashMap, path::PathBuf};
 
 use super::RepositoryId;
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct Config {
+    /// Console address.
+    pub console_address: String,
+    /// Container UID
+    pub container_uid: u32,
+    /// Container GID
+    pub container_gid: u32,
+    /// Directory with unpacked containers.
+    pub run_dir: PathBuf,
+    /// Directory where rw data of container shall be stored
+    pub data_dir: PathBuf,
+    /// Directory for logfile
+    pub log_dir: PathBuf,
+
+    pub repositories: HashMap<RepositoryId, Repository>,
+    pub cgroups: CGroups,
+    pub devices: Devices,
+
+    /// Debugging options
+    pub debug: Option<Debug>,
```

Comment:
why optional? you already can enable/disable individual sections

Reply:
I wanted to avoid having a struct filled with `Nones`.